clang/test/AST/Interp/arrays.cpp

```diff
@@ -352,10 +352,6 @@ namespace ZeroInit {
 }
 
 namespace ArrayInitLoop {
-  /// FIXME: The ArrayInitLoop for the decomposition initializer in g() has
-  /// f(n) as its CommonExpr. We need to evaluate that exactly once and not
-  /// N times as we do right now.
```

Comment:
This comment needs to stay.

Reply:
Oh, right, `Interp` still misevaluates it.